File: src/game/GravityGolf.java
package game;

/**
 * Top-level launching for the game.
 * @author Sean Lewis
 */
final class GravityGolf {

	/** Game log tracker. Outputs to logs\gamelog.txt */
	DataHandler DataWriter = new DataHandler();

	/** Launches the game. */
	public static void main(String[] args) {
		new GameFrame();
	}

}

File: src/main/scala/game/GravityGolf.scala
package game

/**
 * Top-level launching for the game.
 * @author Sean Lewis
 */
object GravityGolf {

  /** Game log tracker. Outputs to logs\gamelog.txt */
  val DataWriter: DataHandler = new DataHandler

  /** Launches the game. */
  def main(args: Array[String]): Unit = {
    new GameFrame()
  }

}
